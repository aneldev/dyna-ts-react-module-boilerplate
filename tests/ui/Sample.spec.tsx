import "jest";

import {configure} from 'enzyme';
import * as Adapter from 'enzyme-adapter-react-16';

configure({adapter: new Adapter()});

import * as React from 'react';
import * as enzyme from 'enzyme';

describe('Sample UI test', () => {
  let wrapper;

  test('with minimal props', () => {
    wrapper = enzyme.shallow(
      (
        <div onClick={() => console.log('I am clicked')}>Demo empty div</div>
      ),
      {},
    );

    expect(wrapper).toMatchSnapshot();
  });
});
