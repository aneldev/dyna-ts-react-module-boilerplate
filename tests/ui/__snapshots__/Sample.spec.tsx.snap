// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Sample UI test with minimal props 1`] = `
<div
  onClick={[Function]}
>
  Demo empty div
</div>
`;
